webapp-django/crashstats/supersearch/tests/test_form_fields.py

```diff
@@ -154,3 +156,32 @@ def test_several_fields_illogically_datetimefield(self):
             field.clean,
             ['<2016-08-01', '<2016-08-02', '<2016-08-03']
         )
+
+    def test_boolean_field(self):
+        field = form_fields.BooleanField(required=False)
+        # If the input is None, leave it as None
+        cleaned_value = field.clean(None)
+        eq_(cleaned_value, None)
+
+        # The list of known truthy strings
+        for truthy in ('__TRUE__', 'true', 't', '1', 'Y', 'Yes'):
+            cleaned_value = field.clean(truthy)
+            eq_(cleaned_value, '__true__')
+
+        # Not truthy strings
+        for truthy in ('__False__', 'False', 'f', '0', 'n', 'No'):
```

Comment:
I don't think I'd call this variable `truthy` but rather `data` or `text` or something. It's a little confusing.

Also, I'd check for empty string and maybe `None` just so we know what happens with those in case this gets used elsewhere or copied or something.